rfcs/023-images-endpoint/README.md

```diff
@@ -0,0 +1,48 @@
+# RFC 023: Images endpoint
+
+**Status:** :construction: Draft :building_construction:
+
+**Last updated:** 27/04/2020
+
+## Context and requirements
+
+- Images are now entities in the catalogue API
```

Comment:
Might be worth being more specific about what "images" are in this context, to be clear that this is not about digitised content a la IIIF. That they are more conceptual, like works, and consist of images from the visual collections and interesting images from inside books and other materials.